app/routes/joblistings/components/JoblistingEditor.js

```diff
@@ -24,195 +27,249 @@ type Props = {
-          name="responsible"
-          component={SelectInput.AutocompleteField}
-          filter={['companies.companycontact']}
-        />
-        <Field
-          name="description"
-          className={styles.descriptionField}
-          label="Søknadsintro"
-          placeholder="Intro til søknaden"
-          component={EditorField.Field}
-        />
-        <Field
-          name="text"
-          className={styles.textField}
-          label="Søknadstekst:"
-          placeholder="Hoveddel av søknaden"
-          component={EditorField.Field}
-        />
-
-        <Flex
-          className={styles.buttonRow}
-          alignItems="baseline"
-          justifyContent="flex-end"
-        >
-          {!isNew && (
-            <Button dark onClick={() => deleteJoblisting(joblisting.id)}>
-              Delete
+            <Button className={styles.submit} submit>
+              Lagre
             </Button>
-          )}
-          <Button className={styles.submit} submit>
-            Lagre
-          </Button>
-        </Flex>
-      </Form>
-    </Content>
-  );
+          </Flex>
+        </Form>
+      </Content>
+    );
+  }
 }
 
-export default reduxForm({
-  form: 'joblistingEditor',
-  enableReinitialize: true,
-  validate(values) {
-    const errors = {};
-    if (!values.title) {
-      errors.title = 'Du må gi jobbannonsen en tittel';
-    }
-    if (!values.description) {
-      errors.description = 'Du må skrive en søknadsintro';
-    }
-    if (!values.company || values.company.value == null) {
-      errors.company = 'Du må angi en bedrift for jobbannonsen';
-    }
-    if (parseInt(values.fromYear, 10) > parseInt(values.toYear, 10)) {
-      errors.toYear = "'Til år' kan ikke være lavere enn 'Fra år'";
-    }
-    if (!values.workplaces) {
-      errors.workplaces = 'Arbeidssteder kan ikke være tom';
-    }
+const validate = values => {
+  const errors = {};
+  if (!values.title) {
+    errors.title = 'Du må gi jobbannonsen en tittel';
+  }
+  if (!values.description) {
+    errors.description = 'Du må skrive en søknadsintro';
+  }
+  if (!values.company || values.company.value == null) {
+    errors.company = 'Du må angi en bedrift for jobbannonsen';
+  }
+  if (parseInt(values.fromYear, 10) > parseInt(values.toYear, 10)) {
+    errors.toYear = "'Til år' kan ikke være lavere enn 'Fra år'";
+  }
+  if (!values.workplaces) {
+    errors.workplaces = 'Arbeidssteder kan ikke være tom';
+  }
 
-    // TODO: and unify timezone handling
-    // $FlowFixMe
-    const visibleFrom = moment.tz(values.visibleFrom, config.timezone);
+  // TODO: and unify timezone handling
+  // $FlowFixMe
+  const visibleFrom = moment.tz(values.visibleFrom, config.timezone);
 
-    // $FlowFixMe
-    const visibleTo = moment.tz(values.visibleTo, config.timezone);
-    if (visibleFrom > visibleTo) {
-      errors.visibleTo = 'Sluttidspunkt kan ikke være før starttidspunkt';
-    }
-    return errors;
+  // $FlowFixMe
```

Comment:
do you still need this FloxFixMe?

Reply:
Don't think so. Guess it can be removed.